History.md

```diff
@@ -1,5 +1,36 @@
 ## v2.3.0, UNRELEASED
 
+### Breaking changes
+* Removed deprecated `mobile-port` flag
```

Comment:
these 3 should be under `meteor-tool@2.3`

Reply:
Done